v2/googlecloud-to-googlecloud/src/test/java/com/google/cloud/teleport/v2/utils/SpannerUtilsTest.java

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright (C) 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.utils;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.cloud.ByteArray;
+import com.google.cloud.Date;
+import com.google.cloud.Timestamp;
+import com.google.cloud.spanner.Struct;
+import com.google.cloud.spanner.Value;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.util.Collections;
+
+/** Unit tests for {@link SpannerUtils} class. */
+@RunWith(JUnit4.class)
+public class SpannerUtilsTest {
+
+  @Test
+  public void testBooleanStructToJson() {
+    JsonObject jsonObject = new JsonObject();
+    jsonObject.addProperty("value", true );
+    assertThat(jsonObject)
+        .isEqualTo(SpannerUtils.convertStructToJson(Struct.newBuilder().set("value").to(Value.bool(true)).build()));
```

Comment:
Please do a static import of `convertStructToJson`, so you don't have to keep adding `SpannerUtils` to the call. It's cleaner that way.